containers/podman-desktop
# fix: use prettier for formatting markdown

### What does this PR do?

Use `prettier` for markdown, instead of `biome`.

Partially reverts 83c9bccd8556a9e13f5cf3eb6575fcafdc152a22

### Screenshot / video of UI



### What issues does this PR fix or reference?

* #10482

### How to test this PR?

`pnpm format:check`


```
Checking formatting...
[warn] website/blog/2024-10-05-kubernetes-blog.md
[warn] website/blog/2024-10-29-creating-an-extension.md
[warn] website/blog/2024-10-31-release-1.14.md
[warn] website/blog/2024-11-14-podman-desktop-cncf.md
[warn] website/blog/2024-11-26-ai-lab-first-app.md
[warn] website/blog/2024-12-12-release-1.15.md
[warn] website/blog/2024-12-16-cncf-projects.md
[warn] website/docs/extensions/index.md
[warn] website/docs/extensions/install/using-extension.md
[warn] website/docs/installation/macos-install.md
[warn] website/docs/installation/windows-install/index.md
[warn] website/docs/kind/installing-extension.md
[warn] website/docs/kubernetes/applying-a-yaml-manifest.md
[warn] website/docs/kubernetes/configuring-editing-kube-object.md
[warn] website/docs/kubernetes/deploying-a-pod-to-kubernetes.md
[warn] website/docs/kubernetes/port-forwarding.md
[warn] website/docs/migrating-from-docker/index.md
[warn] website/docs/migrating-from-docker/managing-docker-compatibility.md
[warn] website/docs/migrating-from-docker/using-the-docker_host-environment-variable.md
[warn] website/docs/minikube/installing-extension.md
[warn] website/docs/podman/creating-a-podman-machine.md
[warn] website/docs/troubleshooting/access-logs.md
[warn] website/docs/troubleshooting/troubleshooting-extension-issues.md
[warn] website/docs/troubleshooting/troubleshooting-podman-on-macos.md
[warn] website/docs/troubleshooting/troubleshooting-podman-on-windows.md
[warn] website/docs/troubleshooting/troubleshooting-podman.md
[warn] website/tutorial/deploying-a-kubernetes-application.md
[warn] Code style issues found in 27 files. Run Prettier with --write to fix.
```
